--- origsrc/fxsudoku-0.5.1/build/configdetect	2006-04-02 09:53:45.000000000 -0500
+++ src/fxsudoku-0.5.1/build/configdetect	2007-01-28 00:03:41.515625000 -0600
@@ -18,7 +18,7 @@
 	else
   	CONFIG_FILE="build/config.linux"
   fi
-elif [ $OSNAME = "CYGWIN_NT-5.0"] ; then 
+elif [ $OSNAME = "CYGWIN_NT-5.0" -o $OSNAME = "CYGWIN_NT-5.1" ] ; then 
   CONFIG_FILE="build/config.cygwin"
 elif [ $OSNAME = "HP-UX" ] && [ -f build/config.hpux ] ; then
 	CONFIG_FILE="build/config.hpux"
